input/index.md

```diff
@@ -83,7 +83,8 @@ These are vulnerabilities that allow an app (malicious or compromised) to either
 ### Press releases
 
  * [Thursday 8th October 2015](press/2015-10-08): 87% of Android devices insecure - Manufacturers fail to provide security updates
- * [Light Blue Touchpaper](https://www.lightbluetouchpaper.org/2015/10/08/87-of-android-devices-insecure-because-manufacturers-fail-to-provide-security-updates/): 87% of Android devices insecure because manufacturers fail to provide security updates
+ * [Light Blue Touchpaper, Thursday 8th October 2015](https://www.lightbluetouchpaper.org/2015/10/08/87-of-android-devices-insecure-because-manufacturers-fail-to-provide-security-updates/): 87% of Android devices insecure because manufacturers fail to provide security updates
+   * [Light Blue Touchpaper, Wednesday 10th July 2019](https://www.lightbluetouchpaper.org/2019/07/10/the-lifetime-of-an-android-api-vulnerability/): The lifetime of an Android API vulnerability
```

Comment:
Better to put this in reverse chronological order with most recent item first.

Reply:
Fixed